dist/api_data.py

```diff
@@ -45,6 +45,45 @@ def __ge__(self, other):
 common_runtime_config = [
     Config('app_metadata', '', r'''
         application-owned metadata for this object'''),
+    Config('assert', '', r'''
+        enable enhanced checking. ''',
+        type='category', subconfig= [
+        Config('write_timestamp', 'off', r'''
+            verify that commit timestamps are used per the configured
+            \c write_timestamp_usage option for this table. The \c diagnostic
+            option only performs this enhanced checking for diagnostic builds
+            whereas, \c on performs it unconditionally.''',
```

Comment:
Just noticed this. The scripts add a trailing period. So all descriptions should not end with a period.

Reply:
7ae8954